docker/files/network/run.sh.base

```diff
@@ -16,7 +16,7 @@ DISABLE_DOCKER=true yarn truffle migrate
 # Export deployed addresses
 
 ETHER_ROUTER_ADDRESS=$(jq -r .etherRouterAddress etherrouter-address.json)
-MINER_ACCOUNT_ADDRESS=$(jq -r '.addresses | to_entries | .[16] | .value' ganache-accounts.json)
+MINER_ACCOUNT_ADDRESS=$(jq -r '.addresses | to_entries | .[5] | .value' ganache-accounts.json)
```

Comment:
I moved this to the back of the array when selecting the address in order to not mess with the accounts  we use for testing.

It's ok for now, but I'll make sure to remove the 6th account from the available wallets in app, just to avoid confusion.